project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+    "System.Collections.NonGeneric": "4.0.1",
```

Comment:
We should remove the non-generic collection usage, but wanted to start with minimal churn.